src/reflection/FStar.Reflection.Data.fs

```diff
@@ -39,7 +40,7 @@ type term_view =
     | Tv_Type   of unit
     | Tv_Refine of binder * term
     | Tv_Const  of vconst
-    | Tv_Uvar   of int * typ
+    | Tv_Uvar   of Z.t * typ
```

Comment:
This looks like overkill to me. It should just be an int, since we're only using ints in F# anyway to generate uvar ids.

Reply:
Although I suppose it makes the code more uniform